README.md

```diff
@@ -205,6 +205,50 @@ helm repo update
 helm upgrade <release-name> galasa/ecosystem --reuse-values --set galasaVersion=0.33.0 --wait
 ```
 
+### Rotating Encryption Keys
+When the Galasa Ecosystem Helm chart is installed, a Kubernetes Secret is created which contains the base64-encoded, 256-bit encryption keys used to encrypt credentials stored in the Galasa Ecosystem's credentials store using AES-256-GCM encryption.
+
+The encryption keys are stored in the following YAML structure:
+```yaml
+encryptionKey: <base64-encoded-encryption-key>
+fallbackDecryptionKeys:
+- <base64-encoded-encryption-key>
+- <base64-encoded-encryption-key>
+```
+
+The `encryptionKey` key in the YAML entry represents the active base64-encoded, 256-bit encryption key used to encrypt credentials stored in the Galasa Ecosystem's credentials store.
+
+The `fallbackDecryptionKeys` list represents a list of base64-encoded encryption keys that are no longer in use, and allows for encryption keys to be rotated without losing previous encryption keys. This allows encrypted credentials to be decrypted with a fallback decryption key and then be encrypted using the newly activated encryption key.
+
+**Before rotating the encryption keys, it is highly recommended to make a backup of the existing credentials stored in your Galasa Ecosystem by running the following command using the Galasa CLI tool:**
+
+```console
+galasactl secrets get --format yaml > /path/to/backup/file.yaml
+```
+
+**where `/path/to/backup/file.yaml` is either an absolute or relative path of your choice to a file where the backup will be stored.**
+
+To simplify the process of rotating encryption keys and re-encrypting credentials, you can run the [`rotate-encryption-keys.sh`](./rotate-encryption-keys.sh) script via the command-line. This script requires the following command-line utilities to be installed:
+
+- kubectl (v1.30.3 or later)
+- openssl (3.3.2 or later)
+- galasactl (0.38.0 or later)
+
+The following flags can be supplied when running the script:
+- `--release-name <name>` : **Required**. The helm release name provided when installing the ecosystem helm chart (see [Installing your Galasa Ecosystem](#installing-your-galasa-ecosystem))
+- `--namespace <namespace>` : Optional. The Kubernetes namespace where your Galasa Ecosystem is installed
+- `--bootstrap <bootstrap-url>` : Optional. The bootstrap URL of the Galasa ecosystem that is being serviced. Not required if the GALASA_BOOTSTRAP environment variable is set and is pointing to the correct bootstrap URL. Overrides the existing GALASA_BOOTSTRAP environment variable value if set
+
+For example:
+
+```bash
+./rotate-encryption-keys.sh --release-name example --namespace galasa-dev
+```
```

Comment:
How will Windows users rotate encryption keys?

Added manual steps to https://github.com/galasa-dev/galasa.dev/pull/840

Reply:
I think if we don't describe it manually, then if the script fails part way through, can it recover ?
eg: If some secrets use the old, and some use the new after a network outage or something...
Is the script resilient enough to fail at any point, and recover the next time it runs ?

Reply:
Thanks, added manual steps to the README in https://github.com/galasa-dev/helm/pull/54/commits/80bd47cc000a62b6e157afba979106bd91a87687